examples/job/get-file-from-job.py

```diff
@@ -62,32 +60,29 @@
 from exabyte_api_client.endpoints.bank_workflows import BankWorkflowEndpoints
 from exabyte_api_client.endpoints.raw_properties import RawPropertiesEndpoints
 
+
 # ### Create and submit the job
 # 
 # For this job, we'll use the workflow located [here](https://platform.exabyte.io/analytics/workflows/84DAjE9YyTFndx6z3).
 # 
-# This workflow is a single-point total energy calculation using Density-Functional Energy as-implemented in Quantum
-# Espresso version 5.4.0.
+# This workflow is a single-point total energy calculation using Density-Functional Energy as-implemented in Quantum Espresso version 5.4.0.
 # 
 # The PBE functional is used in conjunction with an ultrasoft pseudopotential and a planewave basis set.
 # 
-# The material we will investigate is elemental [Silicon](https://materialsproject.org/materials/mp-149/), as-is from
-# Materials Project.
+# The material we will investigate is elemental [Silicon](https://materialsproject.org/materials/mp-149/), as-is from Materials Project.
 # 
-# > <span style="color: orange">Note</span>: This cell uses our API to copy the unit cell of silicon from Materials Project
-# into your account. It then copies a workflow to get the total energy of a system using Quantum Espresso to your account.
-# Finally, a job is created using the Quantum Espresso workflow for the silicon unit cell, and the job is submitted to the
-# cluster. For more information, please refer to our [run-simulation-and-extract-properties](
-# ./run-simulations-and-extract-properties.ipynb) notebook, located in this directory.
+# > <span style="color: orange">Note</span>: This cell uses our API to copy the unit cell of silicon from Materials Project into your account. It then copies a workflow to get the total energy of a system using Quantum Espresso to your account. Finally, a job is created using the Quantum Espresso workflow for the silicon unit cell, and the job is submitted to the cluster. For more information, please refer to our [run-simulation-and-extract-properties](./run-simulations-and-extract-properties.ipynb) notebook, located in this directory.
```

Comment:
I thought we wanted to cap the line width to 120

Reply:
Black (used by nb_black) along with some other pep8 formatters I've found so far, does not cap linewidths of comments, their general philosophy being they can be syntactically significant (certainly not true in our case)

We can cap linewidth to 120, and here is what I believe our options are:
1. Adding logic in config.py to automatically wrap comment lines (as we were doing in the previous iteration). The time-consuming portion will be figuring out how to automatically apply this to the notebooks in addition to .py files.
2. Using something like https://jupyterlab-code-formatter.readthedocs.io/en/latest/ , which is more robust, but will add more complexity to the user's setup (either in the form of an install.sh or install.bat, or extra commands to run) and additional dependencies to install